### ISSUE TITLE
[VS Extension] Package not found

### ISSUE THREAD

**Issue by Fenny** [1pkg/gopium#11]
Hi @1pkg, awesome package!

But I can't get the package working (both CLI and VS Extension), do you have any suggestions?

![image](https://user-images.githubusercontent.com/25108519/97957148-b8fa7680-1daa-11eb-8a89-40c182ad83fc.png)
```go
gopium 🌺: C:\Users\Work\go\bin\gopium.exe -p c:\Users\Work\Documents\GitHub\fiber -r ^Ctx$ -c gc -a amd64 -l 64 -l 64 -l 64 -d -b -i 0 -w 8 -t 0 ast_go fiber filter_pads memory_pack explicit_paddings_type_natural cache_rounding_cpu_l1_discrete
Error: visiting error happened package "fiber" wasn't found at "c:\\Users\\Work\\Documents\\GitHub\\fiber"
// ...
// ...
2020/11/03 07:58:47 visiting error happened package "fiber" wasn't found at "c:\\Users\\Work\\Documents\\GitHub\\fiber"
```

**Comment by kiyonlin** [comment]
Same error here
![unknown](https://user-images.githubusercontent.com/1214670/97961215-dd2b7700-1ded-11eb-87fb-5e584335ce9b.png)

**Comment by 1pkg** [comment]
@Fenny @kiyonlin thank you for reporting the issue. It comes from strict  package name validation in `parser` and should be fixed now in gopium v1.2.0 specifically in commit https://github.com/1pkg/gopium/commit/1472470020efdbfba1e451819d2e015cb03a7e42. Please pool and try new version.

**Comment by 1pkg** [comment]
will consider this fixed
